secant/Util/DebugMenu.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  DebugMenu.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 15.04.2022.
+//
+
+import SwiftUI
+
+// swiftlint:disable:next private_over_fileprivate strict_fileprivate
+fileprivate struct DebugMenuModifier: ViewModifier {
+    enum DragState {
+        case inactive
+        case pressing
+        case dragging(translation: CGSize)
+    }
+
+    @GestureState var dragState = DragState.inactive
+    var minimumDuration: Double
+    let action: () -> Void
+    
+    func body(content: Content) -> some View {
+        let longPressDrag = LongPressGesture(minimumDuration: minimumDuration)
```

Comment:
what's the best way of making this only available on testflight debug builds but not on App Store Releases?

Reply:
I think for appstore build we need different target and define dedicated compiler condition or maybe "not" define it, (in case DEBUG is not enough).

Could be any flag, like `NONPRODUCTION_CODE` and then

```swift
#if DEBUG && NONPRODUCTION_CODE
// code
#endif
```

Reply:
I created the [issue](https://github.com/zcash/secant-ios-wallet/issues/273)

Reply:
In this specific case we could also remove this file from the App Store target and create another one with modifier that does nothing.